web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -121,7 +121,7 @@
         // Retrieve site information
         // TODO: Add INSPIRE, harvester, ... information
         var catInfo = promiseStart.then(function(value) {
-          var url = $scope.url + 'info?_content_type=json&type=site&type=auth';
+          var url = $scope.url + 'info?_content_type=json&type=site&type=auth&type=config';
```

Comment:
Instead of loading the config twice you should better use gnConfig which contains the type=config info. See https://github.com/geonetwork/core-geonetwork/blob/3.0.x/web-ui/src/main/resources/catalog/components/catalog/CatalogService.js#L523